docs/onboarding/access.md

```diff
@@ -15,6 +15,12 @@ If you do not already have a UWO identity, you will need to contact [WTS](https:
 You can then activate your identity 
 [here](https://identity.uwo.ca/activate-hr;jsessionid=C843C01692B7B246356C75B4F1E2C9D4?0).
```

Comment:
One minor comment (realize this wasn't from this PR). Would it be possible to remove the following ;jsessionid=C843C01692B7B246356C75B4F1E2C9D4?0 from the URL here?

This looks like it should be unique to a specific user and isn't strictly necessary for the url.